wled00/data/settings_sec.htm

```diff
@@ -46,6 +33,12 @@
 			fO.value = '';
 			return false;
 	    }
+		function checkNum(o) {
```

Comment:
Agreed that complexity is prefered but since we do not have HTTPS it makes no sense (well, less sense) to make pin ultra complex. 4 digit pin should provide adequate protection until we add HTTPS.